src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java

```diff
@@ -1286,6 +1372,8 @@ public void outputAfterRecipe_EM() {
 
         // Do other stuff from TT superclasses. E.g. outputting fluids.
         super.outputAfterRecipe_EM();
+
+        currentRecipe = null;
```

Comment:
Already gets set to `null` https://github.com/GTNewHorizons/TecTech/blob/eoh_oc_parallel/src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java#L1122

Why was this added?